app/src/auto_validator/settings.py

```diff
@@ -218,9 +219,13 @@ def wrapped(*args, **kwargs):
     #     'task': "auto_validator.core.tasks.demo_task",
     #     'args': [2, 2],
     #     'kwargs': {},
-    #     'schedule': crontab(minute=0, hour=0),
-    #     'options': {"time_limit": 300},
+    #     'schedule': timedelta(seconds=30),
+    #     'options': {"time_limit": 10000},
     # },
+    "update-validator-status-every-20-seconds": {
+        "task": "manager.tasks.update_validator_status",
+        "schedule": timedelta(seconds=20),  # Every 20 seconds
```

Comment:
```suggestion
        "schedule": timedelta(seconds=60),
```
don't add obvious comments or one day they'll become outdated and confusing